### PR TITLE
Create version_control_system.md

### PR DESCRIPTION
**Pull request by gperu**

### CODE REVIEW

**Comment by dgarijo** [comment]
Page is empty?

File: pages/your_tasks/version_control_system.md [hunk 1]
```diff
@@ -1 +1,78 @@
...
+   * Include configuration files and dependencies in version control
+* Facilitate collaboration:
+   * Use a platform like GitHub or GitLab for easy sharing and collaboration
+   * Implement code review practices to maintain quality
+* Maintain your repository:
+   * Regularly backup your repository
+   * Periodically clean up old branches and review access permissions
+
+## How to cite this page
+Giacomo Peru
```
Comment (dgarijo) [inline pages/your_tasks/version_control_system.md:66]: This is not a citation :)

Comment (gperu) [reply pages/your_tasks/version_control_system.md:66]: How can I add the URL if the  PR hasn't been accepted yet? I don't understand 'date of last access'.

Comment (dgarijo) [reply pages/your_tasks/version_control_system.md:66]: Since this needs to be agreed by the project, please just add "TO DO" (or your name + TO DO)

**Comment by dgarijo** [comment]
thanks @gperu I added small comments. Since this is a draft they can be addressed later, but just to give it a thought

**Comment by dgarijo** [comment]
Merging, we'll address comments at some other point

**Pull request merged**